experiments/simple_functions_cv.ipynb

```diff
@@ -0,0 +1,2524 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        _, ax = plt.subplots(figsize=(10, 4))
</pre><p>It is just a detail but, could you run with <code>figsize=(5,4)</code>? This is for having the same size as the other cross-validation plots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Grupo-de-modelado-probabilista/BART/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>